.github/workflows/go.yml

```diff
@@ -7,16 +7,25 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        go: ["1.23.x", "1.22.x", "1.21.x", "1.20.x", "1.19.x", "1.18.x"]
```

Comment:
Why is this changed? We should add 1.24 to the list.

Reply:
this was an accident.  I had rebase conflicts and checked out the wrong file.  Fixed!